validation_test.go

```diff
@@ -90,17 +111,17 @@ var _ = Describe("ValidationError", func() {
 		}
 		err := lerrors.WrapValidation(nerr)
 		Expect(err).NotTo(BeNil())
-		Expect(err.Error()).To(Equal("validation"))
+		Expect(err.Error()).To(Equal(`validation: "fieldName" failed on [ActualTag]`))
```

Comment:
But, that failed in the tag because?
It would be good to add what motivated the failure by recording the translation.

Based on:
-  [Examples translations](https://github.com/go-playground/validator/blob/v9/_examples/translations/main.go)

Reply:
We shouldn't bother about translation since the `. Error()` should only be used for debug/development. Quoting the `go-playground/validator` docs:

> All information to create an error message specific to your application is contained within the FieldError found within the ValidationErrors array.